OWNERS

```diff
@@ -0,0 +1,9 @@
+reviewers:
+  - geeknoid
+  - mandarjog
+  - Douglas-reid
+  - ZackButcher
+  - sebastienvas
+approvers:
```

Comment:
We need more approvers than just mtail and mjog. @geeknoid I don't see a reason all of the current core contributers can't be approvers.